Include/cpython/pystate.h

```diff
@@ -197,12 +197,12 @@ struct _ts {
 /* WASI has limited call stack. Python's recursion limit depends on code
    layout, optimization, and WASI runtime. Wasmtime can handle about 700
    recursions, sometimes less. 500 is a more conservative limit. */
-#ifndef C_RECURSION_LIMIT
+#ifndef Py_C_RECURSION_LIMIT
```

Comment:
Do users define this?

Reply:
I don't know if it's relevant to support that. It's supposed to be a constant built into Python, no?